File: cs/Iris/ParserUtils/LexParseException/CaseException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class CaseException : BaseException
    {
        public CaseException(int lNum, int cNum)
            : base(lNum, cNum)
        {
            
        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append(base.stringTrace());
            str.Append("case expression appears after default expression");

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/CaseException.java
package com.rokuan.iris.exception;

public class CaseException extends BaseException
{
	public CaseException(int lNum, int cNum)
	{
		super(lNum, cNum);   
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append(super.stringTrace());
		str.append("case Expression appears after default Expression");

		return str.toString();
	}
}
